dclab/definitions/feat_const.py

```diff
@@ -118,6 +118,13 @@
     # Fun fact: If we had decided to compute it from the convex contour,
     # then we would have close to none pixelation effects ¯\_(ツ)_/¯.
     ["volume", "Volume [µm³]"],
+    ["area_um_raw", "Area [µm²] from a raw contour"],
+    ["per_um_raw", "Perimeter [µm] from a raw contour"],
+    ["deform_raw", "Deformation from a raw contour"],
+    ["eccentr_prnc", "Eccentricity from a raw contour"],
```

Comment:
Should we replace this with `eccemtr_prnc_raw` to follow the naming scheme for features from raw contours?

Reply:
I use "eccentr_prnc" because we also define "inert_ratio_prnc" from a raw contour.

Reply:
ok